src/main/java/com/htisolutions/poolref/controllers/SubmitScoreController.java

```diff
@@ -0,0 +1,18 @@
+package com.htisolutions.poolref.controllers;
+
+import org.springframework.stereotype.*;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+@RequestMapping("/submit-score")
+public class SubmitScoreController {
+
+    @RequestMapping("/greeting")
+    public String submitScore(
+            @RequestParam(value="winner") String winner,
+            @RequestParam(value="loser") String loser)
```

Comment:
In addition, if you know who's playing the same. If you know the winner, you automatically know the loser. So this would probably pass a GameId and a UserId. But we haven't designed this yet. So I don't think we should be guessing the implementation